Source/WebCore/html/HTMLAttachmentElement.cpp

```diff
@@ -121,24 +124,46 @@ static const AtomString& attachmentSubtitleIdentifier()
     return identifier;
 }
 
+static const AtomString& attachmentSaveIdentifier()
+{
+    static MainThreadNeverDestroyed<const AtomString> identifier("attachment-save"_s);
+    return identifier;
+}
+
+static QualifiedName saveQualifiedName()
+{
+    return QualifiedName { nullAtom(), "save"_s, nullAtom() };
```

Comment:
I wonder if this should be added to `HTMLAttributeNames.in`? We do that for `subtitleAttr` which is specific to `<attachment>`.

If not, I think we should still name this method as `saveAttr`.

cc @whsieh @hortont424

Reply:
I'd prefer to reserve `HTMLAttributeNames.in` to commonly web platform items, I don't think we need to optimize for `<attachment>`. `subtitleAttr` should be scoped to HTMLAttachmentElement.cpp as well fwiw

Reply:
I think both approaches sound reasonable — but either way we choose, I do think it'd be good to make `subtitleAttr` consistent with this new attribute.

Reply:
I'll change `saveQualifiedName` to `saveAttr` here.
For `subtitleAttr`: I've filed rdar://105697448 (subtitleAttr should be scoped to HTMLAttachmentElement.cpp)